libraries/plugins/apis/account_history_api/account_history_api.cpp

```diff
@@ -37,17 +37,30 @@ DEFINE_API_IMPL( account_history_api_chainbase_impl, get_ops_in_block )
 {
    return _db.with_read_lock( [&]()
    {
+      std::multiset< api_operation_object > tmp_result;
```

Comment:
FC has multiset serialization. Let's change the returned type in `get_ops_in_block_return` to the multiset type to eliminate copying to a vector. When we implement this for rocksdb account history plugin, we can override the comparator to always return true so that it behaves as a queue.